src/anomalib/data/datasets/base/image.py

```diff
@@ -273,23 +270,24 @@ def __getitem__(self, index: int) -> DatasetItem:
         if self.task == TaskType.CLASSIFICATION:
             item["image"] = self.augmentations(image) if self.augmentations else image
         elif self.task == TaskType.SEGMENTATION:
-            # Only Anomalous (1) images have masks in anomaly datasets
-            # Therefore, create empty mask for Normal (0) images.
+            # Create empty mask for:
+            # - Normal samples (label_index = 0)
+            # - Unknown samples (label_index = -1)
+            # Only use mask for anomalous samples (label_index = 1)
             mask = (
                 Mask(torch.zeros(image.shape[-2:])).to(torch.uint8)
-                if label_index == LabelName.NORMAL
+                if label_index in {LabelName.NORMAL, LabelName.UNKNOWN}
```

Comment:
Does it make sense to provide an empty mask as gt_mask in the case of unknown label type? I'm concerned it could lead to some unexpected behaviour when users try to evaluate on the private test set.

Maybe it would be better to provide no mask at all for the unknown label? This would prevent running an evaluation sequence on the private test set. (Not sure if this would lead to any practical issues at runtime).